test/parallel/test-http2-ping-settings-heapdump.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const http2 = require('http2');
+const v8 = require('v8');
+
+// Regression test for https://github.com/nodejs/node/issues/28088:
```

Comment:
Looks like the last `:` is redundant here.

Reply:
I think it makes sense, given that a short explanation of the issue follows?

Reply:
Maybe add a space. I think the link is necessary too. but `:` will make IDEs link the wrong url.